spring-cloud-dataflow-composed-task-runner/src/main/java/org/springframework/cloud/dataflow/composedtaskrunner/StepBeanDefinitionRegistrar.java

```diff
@@ -214,6 +224,20 @@ private ComposedTaskProperties composedTaskProperties() {
 		return collector.getTaskApps();
 	}
 
+	private String getPropertyValue(String key) {
```

Comment:
I'd advice not trying to use this legacy `RelaxedNames`. Btw in my PR #4314 I partially moved these property bindings into `ComposedTaskRunnerStepFactory` so that we would not need to do manual binding in this registrar where we really can autowire anything.

Reply:
I had this in the comments on the 2nd commit.  But they are easily hidden.
```
Not sure how to remove the Relaxed names since the registrar creates its own BeanDefinitionBuilder using the ComposedTaskRunnerStepFactory class.
And thus needs to be given the properties at construction time.
```